README.md

```diff
@@ -7,7 +7,7 @@
 [![License](https://img.shields.io/cocoapods/l/CacheAdvance.svg)](https://cocoapods.org/pods/CacheAdvance)
 [![Platform](https://img.shields.io/cocoapods/p/CacheAdvance.svg)](https://cocoapods.org/pods/CacheAdvance)
 
-A cache that enables the performant persistence of individual messages to disk.
+A performant cache for logging systems. CacheAdvance persists log events 15x faster while utilizing 40% less memory than SQLite.
```

Comment:
the memory brag needs more validation. I ran the tests a few times... and now the results are flipping. Investigating.

Reply:
I wonder if it could vary based on lower-level decisions made by iOS? 🤔

Reply:
Memory also seems like something that could vary much more than write time based on subtle differences in configuration and setup. If determining the memory performance vs SQLite proves non-obvious it could make sense to explore the comparison in a future PR. The time performance wins are a great step in and of themselves 👍

Reply:
I'm actually running these tests on the Mac target to try to avoid potential iOS simulator weirdness. But I agree that the time perf wins are still excellent and worth committing.

@martinmroz is jumping on this PR to help me understand the memory delta, as I may be mis-configuring SQL for a true apples-to-apples test. Would like to do a bit more investigation before I pull the plug on that brag entirely 🙂